docker-compose.yml

```diff
@@ -26,7 +28,7 @@ services:
     ports:
       - 8000:8000
     environment:
-      - DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}
+      - DATABASE_URL=postgres://postgres.ugojraljnvusodkofopv:m9BBOv64a9rtvA2g@aws-0-us-west-1.pooler.supabase.com:6543/postgres
```

Comment:
@AfeezGL You shouldn't add your credentials to a public repo.

Reply:
That was a big miss, I'll just roll the credentials.